iroh/src/client/node.rs

```diff
@@ -54,11 +49,23 @@ where
     }
 
     /// Get the id of this node.
-    pub async fn id(&self) -> Result<NodeId> {
+    pub async fn node_id(&self) -> Result<NodeId> {
         let id = self.rpc.rpc(NodeIdRequest).await??;
```

Comment:
why go over rpc for this? was the previous version (`self.inner.secret_key.public()`) somehow wrong?